### PR TITLE
Publish the Ember Data 2.4 release blogpost

### PR DESCRIPTION
**Pull request by bmac**

### CODE REVIEW

File: source/blog/2016-03-13-ember-data-2-4-released.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+[Ember Data 2.4.0 CHANGELOG](https://github.com/emberjs/data/blob/v2.4.0/CHANGELOG.md).
+
+### Upcoming Features in Ember Data 2.5.beta-1
+
+
+#### `ds-finder-include`
+
+The `ds-finder-include` feature allows an `include` query parameter to
+be specified with using `store.findRecord()` and `store.findAll()` as
+described in [RFC 99](https://github.com/emberjs/rfcs/pull/99). This
```
Comment (mixonic) [inline source/blog/2016-03-13-ember-data-2-4-released.md]: I might also suggest calling out that this is a feature of JSON-API

File: source/blog/2016-03-13-ember-data-2-4-released.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+// reload comments
+post.hasMany('comments').reload();
+```
+
+Thanks to [@pangratz](https://github.com/pangratz) for implementing
+this feature.
+
+
+#### `ds-transform-pass-options
```
Comment (mixonic) [inline source/blog/2016-03-13-ember-data-2-4-released.md]: Missing a closing backtick

**Comment by mixonic** [comment]
:clap:

File: source/blog/2016-03-13-ember-data-2-4-released.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+[@nikz](https://github.com/nikz) has implemented
+[RFC 101](https://github.com/emberjs/rfcs/pull/101) which provides
+more context for `RESTAdapter` and `JSONAPIAdapter` errors in Ember
+Data. Be sure to check out the
+[pull request](https://github.com/emberjs/data/pull/3930) for more
+details.
+
+#### Importing Modules
+
+[@pangratz](https://github.com/pangratz) added public paths to make it easy to import the DS.EmbeddedRecordsMixin and DS.Serializer modules.
```
Comment (pangratz) [inline source/blog/2016-03-13-ember-data-2-4-released.md]: For consistency the classes should be in back ticks...

File: source/blog/2016-03-13-ember-data-2-4-released.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+```
+
+Thanks to [@pangratz](https://github.com/pangratz) for implementing
+this feature.
+
+
+#### `ds-transform-pass-options
+
+The `ds-transform-pass-options` feature allows for smarter transforms
+by passing the options object form `DS.attr([type], [options])` into
```
Comment (pangratz) [inline source/blog/2016-03-13-ember-data-2-4-released.md]: `s/form/from` ?

File: source/blog/2016-03-13-ember-data-2-4-released.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+    var options = attributeMeta.options.markdown || {};
+
+    return marked(serialized, options);
+  }
+});
+```
+
+Thanks to [@pangratz](https://github.com/pangratz) for implementing
+this feature and [@knownasilya](https://github.com/knownasilya) for
+proposing the RFC.
```
Comment (pangratz) [inline source/blog/2016-03-13-ember-data-2-4-released.md]: Tiny nitpick: I think for the sake of consistency the RFC should be linked too https://github.com/emberjs/rfcs/pull/1...

**Comment by pangratz** [comment]
Looks good @bmac, thanks for the write-up!

**Comment by bmac** [comment]
Thanks for the feedback @mixonic and @pangratz.

**Pull request merged**